packages/flutter_tools/lib/src/commands/generate_localizations.dart

```diff
@@ -180,10 +188,16 @@ class GenerateLocalizationsCommand extends FlutterCommand {
             'Localizations.of(context), removing the need for null checking in '
             'user code.'
     );
+    argParser.addFlag(
+      'format',
+      help: 'When specified, the `dart format` command is run after generating the files.'
+    );
   }
 
   final FileSystem _fileSystem;
   final Logger _logger;
+  final Artifacts? _artifacts;
```

Comment:
does this need to be nullable?

Reply:
Yes, since globals.artifacts is nullable.